ml/src/test/scala/frameless/ml/FramelessMlSuite.scala

```diff
@@ -0,0 +1,28 @@
+package frameless
+package ml
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.{SQLContext, SparkSession}
+import org.scalactic.anyvals.PosZInt
+import org.scalatest.FunSuite
+import org.scalatest.prop.Checkers
+
+trait SparkTesting {
```

Comment:
Seems this can be used from `dataset` module. Classes in tests should be accessible because we specify dependency as `"test->test;compile->compile"`